volcli/commands.go

```diff
@@ -122,7 +122,16 @@ var Commands = []cli.Command{
 				Action:      VolumeForceRemove,
 			},
 			{
-				Name:        "remove",
+				Name: "remove",
+				Flags: []cli.Flag{cli.BoolFlag{
+					Name:  "r",
```

Comment:
Please capitalize your usage text. While you're here, it'd be nice to have long options, too.

You can do this by separating the options with a comma or a space in the `Name`, I can't remember now. There should be examples in volcli, volmaster, or volplugin structs.